StitchCoreAdminClient/Sources/StitchCoreAdminClient/AuthProviders/ProviderConfig.swift

```diff
@@ -29,11 +29,11 @@ public enum ProviderConfigs: Encodable {
             self.required = required
             self.name = name
         }
-
+        
```

Comment:
[nit] seems like the linter is not being run in this project. if it's trivial to fix then go for it, but I have a feeling we will need to write a script that will add the lint `Run Script` build task when the xcode project is generated. you should file a ticket if that's the case

Reply:
Going to file a separate ticket for all the linted, before this PR expands even more.